buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy

```diff
@@ -579,3 +594,28 @@ def get_sorted_releases() {
 
     return sortedReleases
 }
+
+def draw_summary_table() {
+    def table = get_summary_table(BUILD_IDENTIFIER)
+
+    if (table) {
+        if (ENABLE_SUMMARY_AUTO_REFRESH) {
+            def actions = manager.build.actions
+            for (int i = 0; i < actions.size(); i++) {
+                 def action = actions.get(i)
+                 if (action.metaClass && action.metaClass.hasProperty(action, "text") && action.text.contains("Downstream Jobs Status")) {
```

Comment:
Can we avoid some script approvals by reducing the amount of conditions here, or will we hit NPEs?

Reply:
NPE